.travis.yml

```diff
@@ -114,9 +119,11 @@ jobs:
     - <<: *test_i386_job
       name: Test i386 1.13
       env: GO_VERSION=1.13
+      go: 1.14  # version for host environment used to go list
```

Comment:
This look off, the test is still named "Test i386 1.13" but we're adding `go: 1.14` to it.

Reply:
The resulting matrix is broken now

```
    - <<: *test_i386_job
      name: Test i386 1.13
      env: GO_VERSION=1.13
      go: 1.14  # version for host environment used to go list
    - <<: *test_i386_job
      name: Test i386 1.14
      env: GO_VERSION=1.14
      go: 1.14  # version for host environment used to go list
    - <<: *test_wasm_job
      name: Test WASM 1.13
      env: GO_VERSION=1.13
    - <<: *test_wasm_job
      name: Test WASM 1.14
      env: GO_VERSION=1.14
```

Reply:
Test is done on Go 1.13 on docker.
Host environment requires go command to get a list of subpackages by `go list`, but old version of `go` command fails because of `crypto/ed25519`.

Reply:
Right. But this is extremely confusing. Looking at the matrix now I have no understanding of what version of Go the tests will actually run as, as they appear to be contradicting each other. We need to improve the documentation on this somehow.